.travis.yml

```diff
@@ -0,0 +1,19 @@
+group: edge
+
+services:
+  - docker
+
+on:
+  branch: master
```

Comment:
hum... I don't think this is taken into account on a global scope by travis, which is a good thing anyway IMO as we want the tests to be run on other branches than master, no?